### ISSUE TITLE
[Feature Request] Better theming options

### ISSUE THREAD

**Issue by StaringLongingly** [HemantKArya/BloomeeTunes#7]
1. Option to make the accent color change depending on the album cover
2. Option to change the UI colors manually

**Comment by HemantKArya** [comment]
Thanks @StaringLongingly for reminding this but right now I'm trying to make this app stable and slowly I will implement theme option too. until then I think it will be good if these projects get some contributors for different features.

**Comment by Its-Tony** [comment]
@aneesh1122 भाई को आपकी जरूरत है।

**Comment by aneesh1122** [comment]
> @aneesh1122 भाई को आपकी जरूरत है।

Flutter ka utna idea nhi h bhai. Just abhi kotlin suru kiya hu wo bhi utna nhi aata.

I'm glad that you introduced this app to me. It looks promising. I'll see what I can do.

**Comment by HemantKArya** [comment]
It's nice to you guys starting to love this project 🥰.

**Comment by mrcodefrost** [comment]
Hi @HemantKArya , I can help with this, ready to implement theming across the entire project with light and dark mode. Let me know and we can discuss

**Comment by HemantKArya** [comment]
Thanx you so much @mrcodefrost for reaching us out. about theming light and dark mode, I guess the bright doesn't fit for this cause the accent color and secondary color. Instead of that I am thinking first we have to plan out what combination of colors could be good for theming option then we can provide them in settings and user will be able to select those theme palettes.
btw I don't have too much of color theory Idea but if you any idea about how it will look good in bright mode then please discuss I am open for bright also just need the relaxing color palette for it.
You can reach me out at Instagram or LinkedIn for personal chat if you want. 😊

**Comment by mrcodefrost** [comment]
I am not a color theory expert but yes have implemented fault tolerant dual theming in flutter many times, We can skip the light theme for now but at least try to manage theme globally in a theme.dart, currently a lot of theming is done in local components. I have messaged you on instagram @HemantKArya

**Comment by HemantKArya** [comment]
Yeah that's true that most of the theming is currently done in local components and it would be really helpful if you can help with that I am ready to discuss any idea you have.
